operations/app/terraform/modules/network/main.tf

```diff
@@ -6,22 +6,22 @@
  */
 
 resource "azurerm_subnet" "cdc_app_subnet" {
-  name                 = var.app_subnet_name
-  resource_group_name  = var.resource_group
-  virtual_network_name = var.cdc_vnet_name
-  address_prefixes     = ["172.17.9.64/28"]
-  enforce_private_link_endpoint_network_policies = true  # true = disable, false = enable; see: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subnet
-  service_endpoints    = [
+  name                                           = var.app_subnet_name
```

Comment:
Is there a setting for associating this subnet with the routing table? I made that change manually in the UI.

Reply:
added